Added thematic guide for the ``manim-voiceover`` plugin (#3049)
ManimCommunity/manim: docs/source/guides/add_voiceovers.rst

###########################
Adding Voiceovers to Videos
###########################

Creating a full-fledged video with voiceovers is a bit more involved than
creating purely visual Manim scenes. One has to use `a video editing
program <https://en.wikipedia.org/wiki/List_of_video_editing_software>`__

> to add the voiceovers after the video has been rendered. This process
> can be difficult and time-consuming, since the scenes and the voiceover
> script tend to change a lot during the development process. Creating a video with
> voiceovers in one or two takes requires a lot of planning and preparation,
> which beginners often not know how to do.

Reviewer: This is a bit too opinionated for me, I suggest to rewrite it slightly:

```suggestion
to add the voiceovers after the video has been rendered. This process
can be difficult and time-consuming, since it requires a lot of planning
and preparation.
```
